appservice/src/createAppService/SiteCreateStep.ts

```diff
@@ -18,21 +18,21 @@ import { AppKind, WebsiteOS } from './AppKind';
 import { IAppServiceWizardContext } from './IAppServiceWizardContext';
 
 export interface IAppSettingsContext {
-    storageConnectionString: string;
-    fileShareName: string;
+    storageConnectionString?: string;
+    fileShareName?: string;
     os: string;
-    runtime: string;
+    runtime?: string;
     aiInstrumentationKey?: string;
 }
 
 export class SiteCreateStep extends AzureWizardExecuteStep<IAppServiceWizardContext> {
     public priority: number = 140;
 
-    private createFunctionAppSettings: ((context: IAppSettingsContext) => Promise<NameValuePair[]>) | undefined;
+    private createSiteAppSettings: ((context: IAppSettingsContext) => Promise<NameValuePair[]>) | undefined;
 
-    public constructor(createFunctionAppSettings?: ((context: IAppSettingsContext) => Promise<NameValuePair[]>)) {
+    public constructor(createSiteAppSettings?: ((context: IAppSettingsContext) => Promise<NameValuePair[]>)) {
```

Comment:
Is `createSiteAppSettings` still optional or can we make it required now? Specifically, the bug is only fixed if you pass in `createSiteAppSettings`, right? So maybe we should enforce that?